core/api/src/graphql/public/types/object/consumer-account.ts

```diff
@@ -34,7 +34,7 @@ import DisplayCurrency from "@/graphql/shared/types/scalar/display-currency"
 
 import { listEndpoints } from "@/app/callback"
 import { IInvoiceConnection } from "@/graphql/shared/types/abstract/invoice"
-import { getQuizzesByAccountId } from "@/app/quiz"
+import { listQuizzesByAccountId } from "@/app/quiz"
```

Comment:
is this on purpose? if we use it in this way we dont use the tracing wrapper

Reply:
not sure to understand the question / point you're making

Reply:
if you dont use the wrapper (`Quiz` import from `@/app`) we will not have the proper tracing.. this technique is used in trigger for example (some cases where we avoid to use it)